package.json

```diff
@@ -38,8 +38,9 @@
     "generate-ntp-super-referrer-mapping-table": "node scripts/generateNTPSuperReferrerMappingTable.js",
     "generate-youtubedown": "node ./scripts/generateYoutubedown.js",
     "generate-user-model-installer-updates": "node scripts/generateBraveAdsResourcesComponentInputFiles.js",
-    "lint": "standard",
+    "lint": "standard lib scripts test",
```

Comment:
explicitly specified directories to lint here, since the added detection script would otherwise cause lint errors (e.g. `MutationObserver` is undefined in Node.js contexts).

There is also the option of moving the script into a new repo. I don't feel strongly either way; it's a single file that likely wouldn't be updated too frequently.